src/melee/lb/lbfile.c

```diff
@@ -48,12 +52,15 @@ bool lbFile_800161A0()
 
 #endif
 
-void func_800161C4(s32 r3, s32 r4, s32 r5, s32 r6, s32 r7, s32 r8)
+void func_800161C4(int arg0, int arg1, HSD_Archive* arg2, int arg3, int arg4,
+                   int arg5)
 {
     cancel = false;
-    HSD_DevComRequest(r3, r4, r5, r6, r7, r8, lbFile_8001615C, 0);
-    do {
-    } while (!lbFile_800161A0());
+    HSD_DevComRequest(arg0, arg1, arg2, arg3, arg4, arg5, lbFile_8001615C, 0);
+
+    do
+        continue;
+    while (!lbFile_800161A0());
```

Comment:
This looks super weird, please keep the brackets on this

Reply:
I'll put both then. Anyway I didn't mean to commit this lol.